static/src/stylesheets/module/commercial/_adverts-capi.scss

```diff
@@ -109,6 +109,10 @@
     .adverts__row > * + *::before {
         content: none;
     }
+
+    .adverts__badge__help {
```

Comment:
Now that you mention it, we could totally use the `.badge` namespace, which is free at the moment.